Website/src/items/content/EdgeWebsite/Data/CMP Data/Testimonies/Alba.yml

```diff
@@ -33,15 +36,15 @@ Languages:
         default\Anonymous
     - ID: "8cdc337e-a112-42fb-bbb4-4143751e123f"
       Hint: __Revision
-      Value: "4000c87f-b106-42a5-b135-ea35b776f13f"
+      Value: "85203702-a7ae-41ef-9b52-513ede880168"
     - ID: "a4eb8a28-879d-41d9-87c9-2ed2f03d0ae4"
       Hint: Logo
       Value: |
-        <image stylelabs-content-id="35890" thumbnailsrc="https://demoedge.sitecoresandbox.cloud/api/gateway/35890/thumbnail" src="https://demoedge.sitecoresandbox.cloud/api/public/content/f380beecb80043c3849830195bb9139e?v=930ff2bc" mediaid="" stylelabs-content-type="Image" alt="Alba.jpg" height="3125" width="3125" />
+        <image stylelabs-content-id="35904" thumbnailsrc="https://demoedge.sitecoresandbox.cloud/api/gateway/35904/thumbnail" src="https://demoedge.sitecoresandbox.cloud/api/public/content/949c003d42af49be962cd99f31eb3efd?v=f6979300" mediaid="" stylelabs-content-type="Image" alt="Striva.jpg" height="3125" width="3125" />
```

Comment:
This is Alba and it has "striva" in its `alt` value. The thumbnail link is also the same as the Striva item. Only the src is different.

Please validate and re-choose the image from DAM is necessary.

Reply:
Striva testimony image does not load as well
![image](https://user-images.githubusercontent.com/2000200/143081778-66d4d334-9da4-4df7-b1d8-82679072f33f.png)